src/Provider/RequestId/ServerRequestIdProvider.php

```diff
@@ -2,23 +2,23 @@
 
 namespace TheRedDot\MonologExtraBundle\Provider\RequestId;
 
-final class ApacheUniqueIdProvider implements RequestIdProviderInterface
+final class ServerRequestIdProvider implements RequestIdProviderInterface
 {
     /**
      * @var string
      */
     private $requestId;
 
-    public function __construct(array $serverData = null)
+    public function __construct(string $serverField, array $serverData = null)
     {
         $requestId = uniqid();
```

Comment:
We should not generate a uniqid if a requestId is already provided by the server.